src/MaaWpfGui/Services/Notification/ServerChanNotificationProvider.cs

```diff
@@ -36,24 +36,29 @@ public ServerChanNotificationProvider(IHttpService httpService)
         public async Task<bool> SendAsync(string title, string content)
         {
             var sendKey = ConfigurationHelper.GetValue(ConfigurationKeys.ExternalNotificationServerChanSendKey, string.Empty);
-            var url = $"https://sctapi.ftqq.com/{sendKey}.send";
+
+            // 根据 sendkey 的前缀选择不同的 API URL
+            var url = sendKey.StartsWith("sctp") 
+                ? $"https://{sendKey}.push.ft07.com/send" 
```

Comment:
```suggestion
                ? $"https://{sendKey}.push.ft07.com/send"  // Server酱3
```

https://github.com/easychen/serverchan-demo/blob/08c3f1fd66c48a1d5386d44b283ceb7b3c716f6c/csharp/Program.cs#L29

链接格式确定无误么

Reply:
两天前修改的新接口
我是三天前写的，三天前的链接格式无误
我现在改